openshift4/templates/backend.dc.yaml

```diff
@@ -335,20 +335,20 @@ objects:
                   memory: ${MEMORY_REQUEST}
               livenessProbe:
                 httpGet:
-                  path: ${BASE_PATH}/health
+                  path: ${BASE_PATH}/health/live
                   port: 5000
                   scheme: HTTP
                 initialDelaySeconds: 160
-                timeoutSeconds: 15
-                periodSeconds: 30
+                timeoutSeconds: 1
+                periodSeconds: 1
               readinessProbe:
                 httpGet:
-                  path: ${BASE_PATH}/health
+                  path: ${BASE_PATH}/health/ready
                   port: 5000
                   scheme: HTTP
                 initialDelaySeconds: 160
```

Comment:
Does it really take 160s for the backend to come up? It's a python runtime so I would expect more like 5s.

Reply:
I had the same question, but I believe there is a reason this has been set to 160 @jon-funk . Can we reduce it?

Reply:
The backend is quite slow to come up, I recall it taking 40-60 seconds minimum back when I made this change. Plus variance from how openshift is doing. If you'd like to try tighten it up though, go for it!

Reply:
I am going to experiment with 80 seconds as a safe middle ground. This will be reviewed going forward